src/librustdoc/html/static/main.js

```diff
@@ -487,14 +487,14 @@ function hideThemeButtonState() {
     }
 
     function getHelpElement(build) {
-        if (build !== false) {
+        if (build) {
```

Comment:
In this case, I think the idea was that if you don't pass the `build` value, by default it'll build the popup. However, all calls to the function pass the value so I guess it's fine to update.